content/local-setup.md

```diff
@@ -0,0 +1,283 @@
+# Local Setup
+
+# Lesson Objectives
+
+- Use basic Solana CLI commands
+- Set up a local environment for Solana program development
+- Use Rust and Solana CLI to deploy a Solana program from your local environment
+
+# TL;DR
+
+- To get started with Solana locally, you’ll first need to install Rust and the Solana CLI
+- Once you have Rust and Solana CLI installed, you’ll be able to build and deploy your programs locally using the `cargo build-bpf` and `solana program deploy` commands.
+
+# Overview
+
+To get started with Solana program development, you’ll need to first install Rust and the Solana CLI. Additionally, if you are on a windows computer, it is recommended to use Windows Subsystem for Linux (WSL) to build your Solana Programs.
+
+## Setup on Windows (with Linux)
+
+### Download Windows Subsystem for Linux (WSL)
+
+Open an **administrator** PowerShell or Windows Command Prompt and check windows version
+
+```bash
+winver
+```
+
+If you are on Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11, run the following command.
+
+```bash
+wsl --install
+```
+
+If you are running an older version of windows, follow the instructions [here](https://docs.microsoft.com/en-us/windows/wsl/install-manual). 
+
+You can read more about installing WSL [here](https://docs.microsoft.com/en-us/windows/wsl/install).
+
+### Download Ubuntu
+
+Next, download Ubuntu [here](https://apps.microsoft.com/store/detail/ubuntu-2004/9N6SVWS3RX71?hl=en-us&gl=US). Ubuntu provides a terminal that allows you to run Linux on a Windows computer. This is where you’ll be running Solana CLI commands.
+
+### Download Rust (for WSL)
+
+Next, open an Ubuntu terminal and download Rust for WSL using the following command. You can read more about downloading Rust [here](https://www.rust-lang.org/learn/get-started).
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+### Download Solana CLI
+
+Now we are ready to download Solana CLI for Linux. Go ahead and run the following command in an Ubuntu terminal. You can read more about downloading Solana CLI [here](https://docs.solana.com/cli/install-solana-cli-tools). 
+
+```bash
+sh -c "$(curl -sSfL https://release.solana.com/v1.10.24/install)"
+```
+
+## Solana CLI Basics
+
+Let’s go over some basic Solana CLI commands.
+
+First, open a terminal and check the version of Solana CLI by running the `solana --version` command.
+
+```bash
+solana --version
+```
+
+Next, to see the current Solana CLI configuration settings, run the `solana config get` command.
```

Comment:
Can we explain the configuration settings a big more? Like "The Solana CLI stores a number of configuration settings that impact the behavior of certain commands." Then maybe list a table of config values 🤷 

Then "you can use the following command to view the current configuration:" followed by the `solana config get` command that's there.